Documentation/contributing/testing/e2e.rst

```diff
@@ -212,6 +217,35 @@ has been resolved):
 
 Finally, you can use the instructions from :ref:`the previous chapter<test_cilium_on_lvh>` to run and to test Cilium.
 
+Network performance test
```

Comment:
I believe that this is useful to check the connectivity between nodes in transport layer including performance result for TCP and UDP.

Reply:
LGTM. Thank you for your contribution.